src/index.html

```diff
@@ -21,7 +21,411 @@
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__name-price">Price:</p>
+            <p class="card__num-price">$2,199</p>
+          </div>
+
+          <button
+            class="card__button"
+            data-qa="card-hover"
+            type="button"
+          >
+            Buy
+          </button>
+        </div>
+
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="imac image"
+          />
+
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__name-price">Price:</p>
+            <p class="card__num-price">$2,199</p>
+          </div>
+
+          <button
+            class="card__button"
+            data-qa="card-hover"
+            type="button"
+          >
+            Buy
+          </button>
+        </div>
+
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="imac image"
+          />
+
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__name-price">Price:</p>
+            <p class="card__num-price">$2,199</p>
+          </div>
+
+          <button
+            class="card__button"
+            data-qa="card-hover"
+            type="button"
+          >
+            Buy
+          </button>
+        </div>
```

Comment:
The content within each `<div class="card">` is repeated multiple times with the same data. This redundancy suggests that the content is not being dynamically generated. Consider using a templating engine or JavaScript to dynamically populate these cards with unique data to improve maintainability and scalability.